specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2018-12-20/bms.json

```diff
@@ -1828,14 +1828,29 @@
             "Full",
             "Log",
             "Differential",
-            "Incremental"
+            "Incremental",
+            "SnapshotFull"
           ],
           "type": "string",
           "x-ms-enum": {
             "name": "RestorePointType",
             "modelAsString": true
           },
           "readOnly": true
+        },
+        "recoveryPointTierDetails": {
+          "description": "Recovery point tier information.",
+          "type": "array",
```

Comment:
The breaking change tool may not be picking up changes in polymorphic types correctly... these are all breaking changes to an existing API-version and should go through breaking change review.

@njuCZ can you check if OAD is doing the right thing with polymorphic types?  Adding new properties to an existing version should always cause a break, yes?

Reply:
These are the optional fields added to existing version that should not cause break to the client.

Reply:
The official stance of the breaking change folks is no changes to existing api-versions.  Please get their approval.  Process is in the PR description

Reply:
In the PR description, the breaking change checklist is as follows, and none of these changes are included in this PR.

>  Removing API(s) in stable version
>  Removing properties in stable version
>  Removing API version(s) in stable version
>  Updating API in stable version with Breaking Change Validation errors
>  Updating API(s) in preview over 1 year

Furthermore, the description mentions below, which is what we are doing in this PR.

> “adding a new property” no more require ARM API review.

Reply:
Hi @pilor, given that the guidance in the PR description (quoted above) seems to indicate that this change should not need a breaking change review, is it still required to go through the process?

Reply:
Yes, these are breaking changes

Reply:
I have removed all changes to older API versions from this PR.

Reply:
I have removed all changes to previous versions from this PR. This PR now contains only the new version addition.